utils/swift_build_support/swift_build_support/targets.py

```diff
@@ -152,6 +152,34 @@ def __init__(self, platform, arch):
     def name(self):
         return "{}-{}".format(self.platform.name, self.arch)
 
+    def swift_flags(self, args, resource_dir_root=""):
+        flags = ''
+        if self.platform.name == 'android':
```

Comment:
If this is specific to the android platform, can you add this as a helper on that class?

Reply:
Sure.

Reply:
I changed these to methods on `AndroidPlatform` and ran the build again: only after it errored did I remember why I added these methods to this `Target` class instead, ie `Target` has an arch whereas `Platform` just has a list of possible arches. Since these flags are generated for a single architecture, I need to keep them here.

Reply:
is it possible to pass this in to a method on platform... IDK. My worry here is that this will lead us to a build-script-impl world where we have a large if else depending on what we are building for?

Reply:
Yes, but I would have to pass in the selected arch as a parameter to `swift_flags()`. I have that refactor patch lying around, so I will add the arch part and push it, then you can decide if it's what you want.

Reply:
Made this change in latest commit.